Missing editor destroy in some of tests

This causes https://github.com/ckeditor/ckeditor5-engine/issues/937 what's kinda weird.

https://github.com/ckeditor/ckeditor5-engine/issues/937 is not the only problem. 

Running engine tests right after typing fails one test case.
```
gulp test --files typing,engine
```
```
FAILED TESTS:
  EditingController
    conversion
      ✖ should convert selection from view to model
        Chrome 58.0.3029 (Mac OS X 10.12.4)
      Error: Uncaught AssertionError: expected '<paragraph>[]foo</paragraph><paragraph></paragraph><paragraph>bar</paragraph>' to equal '<paragraph>foo</paragraph><paragraph></paragraph><paragraph>b[a]r</paragraph>' (node_modules/chai/chai.js:206)
```

There might be more similar situations but "luckily" typing is at the end of the repositories list and typing tests are run as one of last.

--- a/tests/delete.js
+++ b/tests/delete.js
@@ -20,6 +20,10 @@ describe( 'Delete feature', () => {
 			} );
 	} );
 
+	afterEach( () => {
+		return editor.destroy();
+	} );
+
 	it( 'creates two commands', () => {
 		expect( editor.commands.get( 'delete' ) ).to.have.property( 'direction', 'backward' );
 		expect( editor.commands.get( 'forwardDelete' ) ).to.have.property( 'direction', 'forward' );

--- a/tests/deletecommand-integration.js
+++ b/tests/deletecommand-integration.js
@@ -35,6 +35,10 @@ describe( 'DeleteCommand integration', () => {
 			} );
 	} );
 
+	afterEach( () => {
+		return editor.destroy();
+	} );
+
 	function assertOutput( output ) {
 		expect( getData( doc ) ).to.equal( output );
 	}

--- a/tests/deletecommand.js
+++ b/tests/deletecommand.js
@@ -14,7 +14,7 @@ describe( 'DeleteCommand', () => {
 	testUtils.createSinonSandbox();
 
 	beforeEach( () => {
-		return ModelTestEditor.create( )
+		return ModelTestEditor.create()
 			.then( newEditor => {
 				editor = newEditor;
 				doc = editor.document;
@@ -24,6 +24,10 @@ describe( 'DeleteCommand', () => {
 
 				doc.schema.registerItem( 'p', '$block' );
 			} );
+	} );
+
+	afterEach( () => {
+		return editor.destroy();
 	} );
 
 	it( 'has direction', () => {

--- a/tests/input.js
+++ b/tests/input.js
@@ -54,6 +54,10 @@ describe( 'Input feature', () => {
 			} );
 	} );
 
+	after( () => {
+		return editor.destroy();
+	} );
+
 	beforeEach( () => {
 		editor.setData( '<p>foobar</p>' );
 

--- a/tests/inputcommand.js
+++ b/tests/inputcommand.js
@@ -19,7 +19,7 @@ describe( 'InputCommand', () => {
 	testUtils.createSinonSandbox();
 
 	before( () => {
-		return ModelTestEditor.create( )
+		return ModelTestEditor.create()
 			.then( newEditor => {
 				editor = newEditor;
 				doc = editor.document;
@@ -32,6 +32,10 @@ describe( 'InputCommand', () => {
 				doc.schema.registerItem( 'p', '$block' );
 				doc.schema.registerItem( 'h1', '$block' );
 			} );
+	} );
+
+	after( () => {
+		return editor.destroy();
 	} );
 
 	beforeEach( () => {

--- a/tests/spellchecking.js
+++ b/tests/spellchecking.js
@@ -20,10 +20,10 @@ import { getData as getModelData } from '@ckeditor/ckeditor5-engine/src/dev-util
 import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view';
 
 describe( 'Spellchecking integration', () => {
-	let editor, onChangesDone;
+	let editor, onChangesDone, container;
 
 	before( () => {
-		const container = document.createElement( 'div' );
+		container = document.createElement( 'div' );
 		document.body.appendChild( container );
 
 		return ClassicEditor.create( container, {
@@ -32,6 +32,12 @@ describe( 'Spellchecking integration', () => {
 		.then( newEditor => {
 			editor = newEditor;
 		} );
+	} );
+
+	after( () => {
+		container.remove();
+
+		return editor.destroy();
 	} );
 
 	beforeEach( () => {

--- a/tests/tickets/59.js
+++ b/tests/tickets/59.js
@@ -27,6 +27,12 @@ describe( 'Bug ckeditor5-typing#59', () => {
 		} );
 	} );
 
+	afterEach( () => {
+		container.remove();
+
+		return editor.destroy();
+	} );
+
 	it( 'editor does not blow up when deleting last styled character', () => {
 		editor.document.enqueueChanges( () => {
 			editor.editing.view.getDomRoot().focus();

